web/package.json

```diff
@@ -14,10 +14,10 @@
   },
   "dependencies": {
     "@radix-ui/react-dialog": "0.1.7",
-    "@redwoodjs/auth": "1.5.1",
-    "@redwoodjs/forms": "1.5.1",
-    "@redwoodjs/router": "1.5.1",
-    "@redwoodjs/web": "1.5.1",
+    "@redwoodjs/auth": "^2.0.0-rc",
+    "@redwoodjs/forms": "^2.0.0-rc",
+    "@redwoodjs/router": "^2.0.0-rc",
+    "@redwoodjs/web": "^2.0.0-rc",
```

Comment:
Could we remove these changes from this PR? I've handled the v2 upgrade already and it threw me off to see an upgrade to v2 in an otherwise innocuous PR about updating README.